.travis.yml

```diff
@@ -7,7 +7,6 @@ node_js:
 - '6.1'
 env:
   matrix:
-  - TEST_BROWSER_NAME=PhantomJS
```

Comment:
It requires you to manually switch over to the browser though, which is a much more cumbersome workflow.

Is it possible to depend on a beta version of PhantomJS? Is it on npm?